pkg/scd/models/availability.go

```diff
@@ -14,8 +12,9 @@ type UssAvailabilityState string
 
 // UssAvailabilityStatus models an uss availability status.
 type UssAvailabilityStatus struct {
-	Uss          dssmodels.Manager
 	Availability UssAvailabilityState
+	Uss          string
```

Comment:
I think giving a strong type hint here with `dssmodels.Manager` is probably clearer -- it is a string underneath, but the additional type indicates to developers what kinds of values we expect in that string.

Less important, but I'd also probably keep the original ordering (`Uss` then `Availability`) -- this is essentially a versioned key-value pair and the Uss field is the key.